src/foam/nanos/export/CSVDriver.js

```diff
@@ -22,10 +22,10 @@ foam.CLASS({
       outputter.outputFObject(X, obj);
       return outputter.toString();
     },
-    function exportDAO(X, dao) {
+    function exportDAO(X, dao, useFilteredColumns) {
```

Comment:
Do we need to add this? Couldn't we just update filteredTableColumns in the context?

Reply:
I thought that if we do that, we'll lose info about columns that was filtered by user, 
but I'll check my assumption

Reply:
fixed